app/services/csv_imports/gias_import.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+module CSVImports
+  class GiasImport
```

Comment:
Can we use the `ServicePattern` here? https://github.com/DFE-Digital/publish-teacher-training/blob/c13e26f96a7aaccd9ceec1305c6aea966a6146f6/app/services/service_pattern.rb#L1  as we're moving to using `.call`

Reply:
Yep